src/components/App.js

```diff
@@ -27,7 +27,7 @@ const App = () => {
           />
 
           <Route exact path="/creatures/:CR" render={(routeProps) => {
```

Comment:
You could destructure here when you are passing in the props `({ match })` if you wanted to, to shorten `routerProps.match.params.CR` into `match.params.CR`.